lerna.json

```diff
@@ -1,9 +1,6 @@
 {
   "version": "4.5.5",
-  "packages": [
-    "packages/*",
-    "examples/*"
-  ],
+  "packages": ["packages/*", "examples/*", "test-apps/*"],
```

Comment:
Don't forget to put those test-apps as private (in their package.json) so that they're not published along with our core packages

Reply:
Nice catch. I've added it to the `template` so test-apps generated will have `private:true` in them.